languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -340,49 +385,29 @@ attribute node_symbol = node            => symbol = (source-text node), source_n
   .
   (_)@right_stmt) {
 
-    ; scopes flow from the left statement to the right one
-    edge @right_stmt.before_scope -> @left_stmt.after_scope
-
+  ; scopes flow from the left statement to the right one
+  edge @right_stmt.before_scope -> @left_stmt.after_scope
 }
 
 ; program, last statement
 (program
   (_)@last_stmt
   .)@prog {
 
-    ; scopes flow from the last statement to the program
-    edge @prog.after_scope -> @last_stmt.after_scope
-
+  ; scopes flow from the last statement to the program
+  edge @prog.after_scope -> @last_stmt.after_scope
 }
 
 (hash_bang_line)@hashbang {
-
-    node @hashbang.before_scope
-    node @hashbang.after_scope
-
-    edge @hashbang.after_scope -> @hashbang.before_scope
-
+  edge @hashbang.after_scope -> @hashbang.before_scope
 }
 
 (comment)@comment {
-
-    node @comment.before_scope
-    node @comment.after_scope
-    node @comment.value
-
-    edge @comment.after_scope -> @comment.before_scope
-
+  edge @comment.after_scope -> @comment.before_scope
 }
 
 
 
-
```

Comment:
I am inclined to re-insert the whitespace here and elsewhere. It's very useful for visually navigating the file, and it's absence makes that much harder at no functional improvement. Given that we lack editor tooling for the TSG language, this sort of purely textual aid is extremely useful.